foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/platform/database/PostgreSQLPlatform.java

```diff
@@ -231,7 +233,7 @@ public void printJavaCollection(List<Expression> items, ExpressionJavaPrinter pr
      * Create the ~ operator.
      * REGEXP allows for comparison through regular expression.
      */
-    public static ExpressionOperator regexpOperator() {
+    private static ExpressionOperator regexpOperator() {
```

Comment:
why decreasing the visibility here?

Reply:
There is no reason to have more than private on platform specific ExpressionOperator instances creators. But I'll leave this untouched.